src/main/java/com/microsoft/sqlserver/jdbc/ISQLServerStatement.java

```diff
@@ -38,7 +38,7 @@
     public String getResponseBuffering() throws SQLServerException;
 
     /**
-     * Retrieves the <code>cancelQueryTimeout</code> property set on this SQLServerStatement object.
+     * Returns the <code>cancelQueryTimeout</code> property set on this SQLServerStatement object.

```

Comment:
Another inconsistency when highlighting properties.

Reply:
I'm not sure what you want to be done here

Reply:
I was just pointing out there are 3 different ways we highlight code.
1. ''
2. ""
3. <code>